BEIMA.Client/src/pages/Building/AddBuildingPage.js

```diff
@@ -0,0 +1,97 @@
+import { useOutletContext, Link } from 'react-router-dom';
+import { IoArrowBack } from "react-icons/io5";
+import { Card, Button, Form, Row, Col } from 'react-bootstrap';
+import { useEffect, useState } from "react";
+import styles from './AddBuildingPage.module.css';
+import FormListWithErrorFeedback from '../../shared/FormList/FormListWithErrorFeedback.js';
+
+
+const AddBuildingPage = () => {
+  // this will be replaced with API call based on selected device type to get the fields
+  const mandatoryBuildingFields = {
```

Comment:
I don't think it's necessary to have these referred to as "mandatory" as these should be the same fields for every building object.

Reply:
Updated the name to allBuildingFields